Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -682,10 +682,10 @@ void SourceBuffer::setActive(bool active)
         m_source->sourceBufferDidChangeActiveState(*this, active);
 }
 
-void SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegment(InitializationSegment&& segment, CompletionHandler<void()>&& completionHandler)
+void SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegment(InitializationSegment&& segment, CompletionHandler<void(RecieveResult)>&& completionHandler)
```

Comment:
Not arguing on the name itself, but it is misspelled.

Reply:
Nice catch!